### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by yeweidaniel**

### CODE REVIEW

**Comment by yeweidaniel** [comment]
@albertshau

File: README.rst [hunk 1]
```diff
@@ -26,7 +26,11 @@ instantiated in a similar manner::
   ApplicationClient appClient = new ApplicationClient(getClientConfig(), getRestClient());
   appClient.list(new NamespaceId("myns"));
 
-If the artifacts being tested are under development, they should be loaded into the target CDAP instance prior to starting the test.
+If the artifacts being tested are under development, they should be loaded into the target CDAP instance prior to starting the test. If these artifacts are loaded into CDAP through the UI, they will exist under ``USER`` scope, so in the tests they should be referred to accordingly.
+
+Debuggin Tests
```
Comment (albertshau) [inline README.rst]: This is specific to IntelliJ, let's rename to:

Debugging in IntelliJ

Comment (yeweidaniel) [reply README.rst]: Updated.

**Comment by yeweidaniel** [comment]
I noticed that plugins need to be in SYSTEM scope instead of USER scope to work, due to IntegrationTestBase.doClear wiping all the artifacts in USER scope before test starts.  

Will update this doc and comment here when ready for another look.

**Comment by yeweidaniel** [comment]
Ready to merge.

**Pull request merged**